config_py/bin/config_py.py

```diff
@@ -16,6 +15,12 @@
     os.path.dirname(os.path.realpath(__file__)), '..', 'fixtures', 'module'
 )
 
+CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
+
+exec(open(os.path.join(
```

Comment:
@pzmudzinski just asking, but is possible to just `import version` instead of doing this `exec`?

Reply:
This is for getting __version__ global variable, import wouldn't work here

Reply:
do we really need it **global**? :)

Reply:
https://www.python.org/dev/peps/pep-0396/
"When a module (or package) includes a version number, the version SHOULD be available in the __version__ attribute."